Package.swift

```diff
@@ -11,7 +11,7 @@ let package = Package(
             targets: ["Clibunwind", "CSwiftBacktrace", "SwiftBacktrace"])
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-nio.git", from: "1.11.0")
+        .package(url: "https://github.com/apple/swift-nio.git", from: "2.0.0")
```

Comment:
totally random comment but given that you seem to use only `NIOConcurrencyHelpers`, you'd even get away with using `Version("1.0.0") ..< Version("3.0.0")` as the constraint :)

Reply:
that'd allow this package to be used with all NIO 1 and NIO 2 apps

Reply:
Thanks for information!
I opened #19 😄